scripts/create-release-notes.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const path = require('path');
+const prompts = require('prompts');
+const fs = require('fs');
+const getArgs = require('./helpers/get-args');
+const checkComponentApi = require('./check-components-api');
+const prevVersion = require('../package.json').version;
+
+const args = getArgs();
+const pathReleasesRoot = path.resolve(__dirname, '../', 'docs/releases');
+const updatePackageJson = (value) => {
+  const filePath = path.resolve(__dirname, '../package.json');
+  const content = JSON.parse(fs.readFileSync(filePath), 'utf8');
+  content.version = value;
+  fs.writeFileSync(filePath, JSON.stringify(content, null, 2));
+  console.log('🚀 Version updated successfully.');
+};
+
+(async () => {
+  const choices = [
+    'major',
+    'minor',
+    'patch',
+  ];
+  let choiceIndex = choices.findIndex((el) => el === args.releaseType);
+  if (choiceIndex < 0) {
+    choiceIndex = (await prompts([ {
+      type: 'select',
+      name: 'selected',
+      message: 'What type of release do you want to create?',
+      choices,
+    } ])).selected;
+  }
+  const [
+    major,
+    minor,
+    patch,
+  ] = prevVersion.split('.').map(
+    (value, index) => {
+      if (index === choiceIndex) return parseInt(value, 10) + 1;
+      if (index > choiceIndex) return 0;
+      return value;
+    },
+  );
+  const newVersion = `${major}.${minor}.${patch}`;
+  const pathNewVersion = `/v${major}.x.x/v${major}.${minor}.x/v${newVersion}`;
+  const pathVersionRoot = `${pathReleasesRoot}${pathNewVersion}`;
+  const getFilePath = (name) => `${pathVersionRoot}/${name}.stories.mdx`;
+  const isFileExist = (name) => fs.existsSync(getFilePath(name));
+  const diffs = await checkComponentApi(`release/v${prevVersion}`, null, null, `${newVersion}`);
+  const date = new Date();
+  const content = {
```

Comment:
@ArkadiuszKrzymienski maybe we can use templates like in `create-components` script. You can add `release-notes-template` and put to it `changelog.stories.mdx` and `migration-guide.stories.mdx`. Maybe it can be easiest to maintain

Reply:
WDYT?